Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -129,11 +131,25 @@ const BlobReader& VolumeGC::GetBlobReader() const
   return *m_reader;
 }
 
+std::unique_ptr<BlobReader>& VolumeGC::GetBlobReaderPtr()
+{
+  return m_reader;
+}
+
 Platform VolumeGC::GetVolumeType() const
 {
   return Platform::GameCubeDisc;
 }
 
+bool VolumeGC::IsTriforceGame() const
```

Comment:
I know nothing about this code but it seems a bit odd to me that this exists in the `VolumeGC` class.  Would be nice if this could somehow be a static on the `VolTri` class instead.

Reply:
Should be possible, that would mean Gamecube games are assigned to VolumeTri by default, and it would also make things uglier when I implement boot.id parsing. This code isn't mine though, so I'm not 100% sure how to go about it

Reply:
Well, I was hoping you could do something like:

```
if VolumeTri::IsTriforceGame(reader)
  return std::make_unique<VolumeTri>(std::move(reader));
  else
   return std::make_unique<VolumeGC>(std::move(reader));
```

That way keeping all the Trifoce logic separated in its own volume class and not polluting the Gamecube stuff.

But, honestly, I don't really know much about this code so it'd be better to wait for someone else to comment on it :)

Reply:
It cant be static because it uses `this`, but I've moved it to VolumeTri